### PR TITLE
#58 logging with request id

### PR DESCRIPTION
**Pull request by alex-a-renoire**

### CODE REVIEW

**Comment by DrIhor** [comment]
Why we need a separate service to log in and not use **logrus**???

File: service/loggingService/service.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const ContextKeyRequestID ContextKey = "requestID"
+
+type LoggingService struct {
+}
+
+func New() LoggingService {
+	return LoggingService{}
+}
+
+func (h LoggingService) WriteLog(ctx context.Context, message string) {
```
Comment (romanilchyshyn) [inline service/loggingService/service.go:21]: Function signature looks like it takes ctx not to log it but for cancelation.

File: service/loggingService/service.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+)
+
+type ContextKey string
+
+const ContextKeyRequestID ContextKey = "requestID"
+
+type LoggingService struct {
```
Comment (romanilchyshyn) [inline service/loggingService/service.go:14]: I don't see the reason why we need this as a service and not as a helper func in controller.

Comment (alex-a-renoire) [reply service/loggingService/service.go:14]: In two controllers, as well as in the handler. I think it's better like this

File: service/loggingService/service.go (hunk 1 above)
Comment (alex-a-renoire) [reply service/loggingService/service.go:21]: what should i do with this comment?

**Comment by alex-a-renoire** [comment]
> Why we need a separate service to log in and not use **logrus**???

I use logrus inside the service.

**Comment by alex-a-renoire** [comment]
It's needed to extract contexts each time

**Review by DrIhor — APPROVED** [review]

**Review by stasBigunenko — APPROVED** [review]

**Review by romanilchyshyn — APPROVED** [review]

**Pull request merged**